packages/vite-plugin-svelte/src/utils/compile.js

```diff
@@ -59,6 +59,7 @@ export function createCompileSvelte() {
 		/** @type {import('svelte/compiler').CompileOptions} */
 		const compileOptions = {
 			...options.compilerOptions,
+			hmr: options.compilerOptions.hmr && !normalizedFilename.includes('/node_modules/'),
```

Comment:
TS doesn't like this. It sees that `options.compilerOptions` could potentially be undefined.

Reply:
close enough to false :dagger:  thanks!

Reply:
No, that would be an actual issue, I think. If `options.compilerOptions` is undefined, then `options.compilerOptions.hmr` is a runtime error.